### DISCUSSION TITLE
Power: maximum vs allocated

### DISCUSSION THREAD

**Discussion by kav2k** [netbox-community/netbox#7410] [Q&A] [answered]
What are the intended semantics for maximum/allocated power? Can you give a couple of examples? Does it ever make sense to have one but not the other?

Am I correct in thinking that, for a PSU, one would put its rating as maximum, and observed/estimated typical usage in allocated?

**Comment by bitcollector1** [comment, marked as answer]
Allocated power is your budget, what you expect the device to pull under your specific operating parameters. For example our new SONIC switches have a 1900 watt power supply (max) and when we fully populated the device with (32) 400G optics and did a snake test we observed to power supply at about ~900 watts (backup power supply was off).  With DAC cables we saw power supply at ~375 watts, so we set this as our "allocation".   Hope that helps somewhat.

**Reply by signifywinter** [reply]
If you had a UPS or PDU, would you rate the Maximum at the stated rating, or 80% of that rating which is the practical maximum?

Seems like either would work, so long as consistency is maintained.
